packages/field-base/src/slot-target-mixin.js

```diff
@@ -38,28 +40,64 @@ const SlotTargetMixinImplementation = (superclass) =>
     }
 
     /**
-     * Forwards every node from the source slot to the target element
+     * Copies every node from the source slot to the target element
      * once the source slot' content is changed.
      *
      * @private
      */
-    __forwardNodesToSlotTarget() {
+    __checkAndCopyNodesToSlotTarget() {
+      this.__sourceSlotObserver.disconnect();
+
       if (!this._slotTarget) {
         return;
       }
 
+      // Remove any existing clones from the slot target
+      if (this.__slotTargetClones) {
+        this.__slotTargetClones.forEach((node) => {
+          if (node.parentElement === this._slotTarget) {
+            this._slotTarget.removeChild(node);
+          }
+        });
+        delete this.__slotTargetClones;
+      }
+
```

Comment:
Just for my understanding, why is it needed to store and remove slot target clones here despite `_slotTarget.innerHTML` gets cleared later?

Reply:
It only gets cleared if there are non-empty nodes in the source slot. So if you place a text in the source slot and then remove that text, it needs to get removed from the target  also.

Also, we can't clear innerHTML here because it would clear manually added custom label. That's why we need to store references to the clones and only remove them.